amazon-sqs/5.8/modules/ROOT/pages/amazon-sqs-connector-reference.adoc

```diff
@@ -22,9 +21,9 @@ Anypoint Connector for Amazon SQS (Amazon SQS Connector) enables you to interfac
 | Expiration Policy a| <<ExpirationPolicy>> |  Configures the minimum amount of time that a dynamic configuration instance can remain idle before the runtime considers it eligible for expiration. This does not mean that the platform expires the instance at the exact moment that it becomes eligible. The runtime purges the instances when as appropriate. |  | 
 |===
 
-==== Connection Types
+=== Connection Types
 [[config_basic]]
-===== Basic
+==== Basic
 
```

Comment:
Yikes, this is some heaving nesting of info.
I haven't looked at the context, but maybe we need to promote some of this to separate Reference topics.

Reply:
This is a generated referenced. It's just how it gets generated. Can't fix.